### PR TITLE
[FLINK-18382][docs] Translate Kafka SQL connector documentation into Chinese

### PR DESCRIPTION
**Pull request by PatrickRen**
## What is the purpose of the change

This pull request translates Kafka SQL connector documentation into Chinese

## Brief change log

  - Translate Kafka SQL connector documentation into Chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

**Comment by PatrickRen** [comment]
Hi @wuchong , I translated documentation of Kafka SQL connector into Chinese. Can you have a review of this PR? Thanks~

**Comment by wuchong** [comment]
Hi @PatrickRen , the Kafka SQL page has been updated for more information. Do you have time to translate the new parts too?

File: docs/dev/table/connectors/kafka.zh.md [hunk 1]
```diff
@@ -75,185 +72,181 @@ CREATE TABLE kafkaTable (
...
-The config option `topic-pattern`  will use regular expression to discover the matched topic. For example, if the `topic-pattern` is `test-topic-[0-9]`, then all topics with names that match the specified regular expression (starting with `test-topic-` and ending with a single digit)) will be subscribed by the consumer when the job starts running.
+`topic` 和 `topic-pattern` 配置项决定了 source 消费的 topic 或 topic 的匹配规则。`topic` 配置项可接受使用分号间隔的 topic 列表，例如 `topic-1;topic-2`。
+`topic-pattern` 配置项使用正则表达式来探测匹配的 topic。例如 `topic-pattern` 设置为 `test-topic-[0-9]`，则在作业启动时，所有匹配该正则表达式的 topic（以 `test-topic-` 开头，以一位数字结尾）都将被 consumer 订阅。
 
-To allow the consumer to discover dynamically created topics after the job started running, set a non-negative value for `scan.topic-partition-discovery.interval`. This allows the consumer to discover partitions of new topics with names that also match the specified pattern.
+为允许 consumer 在作业启动之后探测到动态创建的 topic，请将 `scan.topic-partition-discovery.interval` 配置为一个非负值，从而使 consumer 能够探测匹配名称规则的新 topic 中的 partition。
```
Comment (V1ncentzzZ) [inline docs/dev/table/connectors/kafka.zh.md:195]: It will be better like "从而使 consumer 能够探测匹配名称规则的 topic 中新的 partitions。"?

**Comment by PatrickRen** [comment]
Hi @wuchong and @V1ncentzzZ , I re-translated Kafka SQL connector's documentation and also fixed a typo in the original English doc. Since the sql-connector-download-table part has been extracted to an individual HTML file in FLINK-20093, I also created a Chinese version for it. Could you have a review on these changes? Thanks~

File: docs/dev/table/connectors/kafka.zh.md (hunk 1 above)
Comment (PatrickRen) [reply docs/dev/table/connectors/kafka.zh.md]: Fixed in the latest commit~

**Comment by wuchong** [comment]
Looks good to me. @PatrickRen , could you also update other Chinese pages to use the Chinese `sql-connector-download-table`?

**Comment by PatrickRen** [comment]
@wuchong I added Chinese version `sql-connector-download-table` for all Chinese SQL docs in the latest commit. Could you have a review again? Thanks~

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**